README.md

```diff
@@ -42,19 +42,32 @@ variable:
     restore: true
   current_power_usage:
     force_update: true
+
+  daily_download:
+    value: 0
+    restore: true
+    domain: sensor
+    attributes:
+      state_class: measurement
+      unit_of_measurement: GB
+      icon: mdi:download
 ```
 
 A variable 'should' have a __value__ and can optionally have a __name__ and __attributes__, which can be used to specify additional values but can also be used to set internal attributes like icon, friendly_name etc.
 
+A variable can accept an optional `domain` which results in the entity name to start with that domain instead of `variable`.
+
 In case you want your variable to restore its value and attributes after restarting you can set __restore__ to true.
 
 In case you want your variable to update (and add a history entry) even if the value has not changed, you can set __force_update__ to true.
 
 ## Set variables from automations
 
-To update a variables value and/or its attributes you can use the service call `variable.set_variable`
+The variable componet exposes 2 services:
```

Comment:
component

Reply:
Fixed it and added code spell (which doesn't catch this but would help with spellings in code).